code/modules/admin/topic.dm

```diff
@@ -1859,7 +1859,7 @@
 		var/role = href_list["editbanrole"]
 		var/duration = href_list["editbanduration"]
 		var/applies_to_admins = text2num(href_list["editbanadmins"])
-		var/reason = href_list["editbanreason"]
+		var/reason = url_decode(href_list["editbanreason"])

```

Comment:
does the implicit `list2params` not already effectively do this?

Reply:
Nope.